docs/ref/request-response.txt

```diff
@@ -833,6 +833,12 @@ Attributes
 
     A bytestring representing the content, encoded from a string if necessary.
 
+.. attribute:: HttpResponse.text
+
+    .. versionadded:: 5.2
+
+    A string representation of :attr:`HttpResponse.content`.
```

Comment:
Thank you for the quick update! A quick second thought (sorry, you were faster than me) is the following: I think it would be useful to provide details of the encoding used to decode the string.

How about:
```suggestion
    A string representation of :attr:`HttpResponse.content`, decoded using the
    response's :attr:`HttpResponse.charset` (defaulting to UTF-8 if empty).
```

Reply:
Yeah I agree this is nicer :+1: